custom_components/tuya_local/devices/gautone_pg103_alarm.yaml

```diff
@@ -0,0 +1,100 @@
+name: GauTone PG-103 alarm
+products:
+  - id: eb26b0818666ecd284o4qe
+    name: GauTone PG-103 alarm security system
+primary_entity:
+  entity: alarm_control_panel
+  dps:
+    - id: 1
+      type: string
+      name: alarm_state
+      mapping:
+        - dps_val: disarmed
+          value: disarmed
+        - dps_val: arm
+          value: armed_away
+        - dps_val: home
+          value: armed_home
+        - dps_val: sos
+          value: triggered
+    - id: 32
+      type: string
+      name: master_state
+    - id: 36
+      type: string
+      name: sub_class
+    - id: 101
+      type: string
+      name: power_state
+secondary_entities:
+  - entity: number
+    name: Exit delay
+    category: config
+    icon: "mdi:timer"
+    dps:
+      - id: 2
+        type: integer
+        name: value
+        unit: sec
+        range:
+          min: 0
+          max: 1000
+  - entity: number
+    name: Entry delay
+    category: config
+    icon: "mdi:timer"
+    dps:
+      - id: 28
+        type: integer
+        name: value
+        unit: sec
+        range:
+          min: 0
+          max: 1000
+  - entity: number
+    name: Siren Duration
+    category: config
+    icon: "mdi:timer"
+    dps:
+      - id: 3
+        type: integer
+        name: value
+        unit: min
+        range:
+          min: 0
+          max: 1000
+  - entity: switch
+    name: Alarm Sound
+    category: config
+    dps:
+      - id: 6
+        type: boolean
+        name: switch
+  - entity: switch
+    name: Backlight
+    category: config
+    dps:
+      - id: 13
+        type: boolean
+        name: switch
+  - entity: switch
+    name: Arm Beep
+    category: config
+    dps:
+      - id: 4
+        type: boolean
+        name: switch
+  - entity: switch
+    name: Mode App Sound
+    category: config
+    dps:
+      - id: 10
+        type: boolean
+        name: switch
+  - entity: switch
+    name: Alarm Message propel

```

Comment:
I think this is for notifications, which may be a more obvious name than "propel".